### DISCUSSION TITLE
Create new Royalty system

### DISCUSSION THREAD

**Discussion by The8thSoul888** [Topkinsme/Color-Battle-Roles#51] [Ideas]
Me and Luffy have briefly discussed this before, but we had an idea to create some sort of new Royalty system, in which more than just Prince may replace the King after they permanently die.

New system:
King <-- Prince <-- Queen <-- Princess

This is the order of replacement. If the King is killed when one of the other roles specified above is alive, the role with the highest priority loses their previous role to become the new King. Any dead Royalty roles with a higher priority than the new King will automatically permanently die (to prevent issues that might arise from the role with the higher priority respawning with the new King in power).

Queen has not been defined as a role at the moment, however we have created an idea for the Queen role: If they die in any other way other than Tribute, the Chief Warrior will have an extra attack the following night.

Also, to reword the Princess, we discussed and came to an agreement that the Princess ability should remain as a one time use, however we believe they should be allowed to be tributed again. (Works like a Prince after their ability has been activated)

Any problems/suggestions? Please, share them. I would like some feedback on this new idea.

**Comment by Topkinsme** [comment]
I like this idea.
